gui/packages/ubuntupro/lib/l10n/app_en.arb

```diff
@@ -35,11 +35,14 @@
     "storeManaged": "This subscription is managed through Microsoft Store.",
     "manageSubscription": "Manage your subscription",
     "orgManaged": "This subscription is owned and managed by your organization.",
-    "manuallyManaged": "Visit {proDashboardLink} to manage your subscription.",
+    "manuallyManaged": "Visit [{proDashboardLink}]({proDashboardLinkHref}) to manage your subscription.",
```

Comment:
We must keep the localizable string as it was and producing the Markdown link syntax in the code, instead of in the translation (since that syntax is independent of l10n), to prevent translators from changing relative positioning of those placeholders by accident, thus breaking the syntax.